packages/vscode-extension/src/builders/customBuild.ts

```diff
@@ -18,9 +17,10 @@ export async function runExternalBuild(
   cancelToken: CancelToken,
   buildCommand: string,
   env: Env,
-  platform: DevicePlatform
+  platform: DevicePlatform,
+  appRoot: string
```

Comment:
nit: in this particular place I'd just call it `cwd` here as the method is pretty abstract, so it doesn't need to have a context that it runs in an app root IMO